### PR TITLE
PR template update

### PR DESCRIPTION
**Pull request by ronny-mysten**
Proposed changes previously discussed. Any issues foreseen?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,21 +2,23 @@
...
+If you check a box other than "None of the above", include information that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 
+
+- [ ] Protocol: 
+- [ ] Nodes (Validators and Full nodes): 
+- [ ] Indexer: 
+- [ ] JSON-RPC: 
+- [ ] GraphQL: 
+- [ ] CLI: 
+- [ ] Rust SDK: 
+- [ ] None of the above
```
Comment (ebmifa) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: Do we really need to have this option? It would be less things to do if we don't list this option?

Comment (ronny-mysten) [reply .github/PULL_REQUEST_TEMPLATE.md:24]: I think the idea was for it to act as an explicit option to let the submitter know that release notes aren't required if this box is checked. I don't feel strongly about its inclusion, so will remove.

Comment (amnn) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yeah, this is only relevant if we also have tooling that prevents lands without one of these options ticked, which is maybe a bit forceful.

**Review by amnn — APPROVED** [review]
LGTM, thanks @ronny-mysten !

**Comment by ebmifa** [comment]
@ronny-mysten can we wait to land this until I update the workflow that validates the template, otherwise shit will break

**Comment by ebmifa** [comment]
> @ronny-mysten can we wait to land this until I update the workflow that validates the template, otherwise, shit will break
@ronny-mysten, actually, never mind, we can land this. The validation workflow on pre-land is content-agnostic, and it should work.

**Review by ebmifa — APPROVED** [review]

**Comment by ebmifa** [comment]
@ronny-mysten One more thing: we should probably announce this first in sui-core to make sure everybody knows about this change and what is expected.

**Pull request merged**